src/element/textWysiwyg.tsx

```diff
@@ -105,6 +105,9 @@ export const textWysiwyg = ({
     const updatedElement = Scene.getScene(element)?.getElement(
       id,
     ) as ExcalidrawTextElement;
```

Comment:
let's also get rid of the `as ExcalidrawTextElemen` — no longer needed and reduces type safety

Reply:
When I removed it I got a build error. Seems it was there for a reason because getElement will return an ExcalidrawElement, but textAlign is only available on ExcalidrawTextElement

Reply:
nvm, I forgot we also do that because of type narrowing. Instead, I've made the `getElement` be generic. What we're after is that the API at least returns a nullable value.